### ISSUE TITLE
ffmpeg failed

### ISSUE THREAD

**Issue by Xarin** [magn2o/UnicornDockerUnofficial#1]
Built using the provided dockerfile
Deployed via
```
---
version: "3"
services:
  plex:
    image: unicorndocker:1.0
    networks:
      - plex-network
    ports:
      - 32400:32400/tcp
      - 3005:3005/tcp
      - 8324:8324/tcp
      - 32469:32469/tcp
      - 1900:1900/udp
      - 32410:32410/udp
      - 32412:32412/udp
      - 32413:32413/udp
      - 32414:32414/udp
      - 3001:3001/tcp
    environment:
      - PLEX_UID=1000
      - PLEX_GID=1000
      - TZ=US/EASTERN
      - PLEX_CLAIM=xxxxxxxxxxxxx
      - PLEX_PATH_SESSIONS=/config/Library/Application Support/Plex Media Server/Cache/Transcode/Sessions
      - DATABASE_SQLITE_PATH=/config/Library/Application Support/Plex Media Server/Plug-in Support/Databases/com.plexapp.plugins.library.db
      - REDIS_HOST=redis
      - REDIS_PORT=6379
    volumes:
      - /docker/volumes/disneyplus.domain.com/config:/config
      - /etc/plex/local:/config/Library/Application Support/Plex Media Server/Plug-in Support
      - /mnt/Media:/mnt/Media
    restart: unless-stopped
  redis:
    image: redis:alpine
    networks:
      - plex-network
    sysctls:
      - net.core.somaxconn=511
    restart: unless-stopped
  tautulli:
    image: linuxserver/tautulli
    networks:
      - plex-network
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=US/EASTERN
    volumes:
      - /etc/tautulli:/config
      - /docker/volumes/disneyplus.domain.com/config/Library/Application Support/Plex Media Server/Logs:/logs
    ports:
      - 8181:8181
    restart: unless-stopped

networks:
  plex-network:
```
Error - (after I have started a stream)
```
Error: SQLITE_CANTOPEN: unable to open database file
Emitted 'error' event at:
npm ERR! code ELIFECYCLE
npm ERR! errno 1
{ FetchError: request to http://127.0.0.1:3001/api/ffmpeg failed, reason: socket hang up
    at ClientRequest.<anonymous> (/snapshot/UnicornFFMPEG/node_modules/node-fetch/lib/index.js:1455:11)
    at ClientRequest.emit (events.js:189:13)
    at Socket.socketOnEnd (_http_client.js:426:9)
    at Socket.emit (events.js:194:15)
    at endReadableNT (_stream_readable.js:1125:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)
  message:
   'request to http://127.0.0.1:3001/api/ffmpeg failed, reason: socket hang up',
  type: 'system',
  errno: 'ECONNRESET',
  code: 'ECONNRESET' }
npm ERR! unicorn-load-balancer@2.0.0 start: `cross-env DEBUG=* node index.js`
npm ERR! Exit status 1
```
The sqlite path exists inside the container:
-rwxrwxrwx 1 plex plex 106M Apr 28 16:27 /config/Library/Application Support/Plex Media Server/Plug-in Support/Databases/com.plexapp.plugins.library.db

Is there anything I missed?
Thanks!

**Comment by magn2o** [comment]
This is likely because I opted to hardcode DATABASE_SQLITE_PATH, assuming (incorrectly) that everyone would be using the standard paths. 

This should be fixed in c23b11acd96734d08eb19968ced6213365b11e3e

**Comment by Xarin** [comment]
Success! After a rebuild and defining SERVER_PUBLIC=https://disneyplus.domain.com and SERVER_PORT=3001 Its working as expected.
Thanks again!
